Route/RoutePageGenerator.php

```diff
@@ -99,9 +99,20 @@ public function update(SiteInterface $site, OutputInterface $output = null)
         }
 
         // Iterate over declared routes from the routing mechanism
+        $routesDedupInfos = array();
+        // We first de-duplicate routes by path; the first declaration win !
         foreach ($this->router->getRouteCollection()->all() as $name => $route) {
-            $name = trim($name);
+            $routePath = $route->getPath();
+            if (array_key_exists($routePath, $routesDedupInfos)) {
+                $this->writeln($output, sprintf('  <info>IGNORE</info> route %s is ignored because it uses the same path as %s (path=%s)',
+                    $name, $routesDedupInfos[$routePath][0], $routePath));
```

Comment:
It looks like [this is part of PSR-2](http://www.php-fig.org/psr/psr-2/#4-4-method-arguments)

> Argument lists MAY be split across multiple lines, where each subsequent line is indented once. When doing so, the first item in the list MUST be on the next line, and there MUST be only one argument per line.